README.md

```diff
@@ -57,6 +57,26 @@ It involves mainly the addition of:
 -   the murano agent configuration file: It includes different murano-agent configuration files for services to be included in the different
   linux distributions.
 
+#### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
+- installing python dependences,
+- generating the core library zip file
+- uploading into murano instance
+
+To execute it, it is required to export admin credentials to access to the Cloud, where Murano is part of.
+
+    $ export OS_USERNAME='admin'
```

Comment:
I do not write 'admin' for user and tenant name and live only the corresponding description like the other.